Objects/Converters/ConverterRevit/ConverterRevitShared/RevitElementTypeUtils.cs

```diff
@@ -140,6 +140,12 @@ public void SetElementFamily(Base @base, string family)
       appObj.Update(logItem: $"Could not find valid incoming type for element of type \"{element.speckle_type}\"");
     }
     var typeInfo = AllCategories.GetRevitCategoryInfo<T>(element);
+
+    if (revitDocumentAggregateCache is null)
+    {
+      return default;
+    }
```

Comment:
Is this safe? This means that it will always return null if the cache is not set, which is not immediately obvious.

Reply:
The logic was around this as below so I thought it is safe to return immediately if even cache is null....

```c#
var types = revitDocumentAggregateCache
      .GetOrInitializeWithDefaultFactory<List<ElementType>>()
      .GetOrAddGroupOfTypes(typeInfo);

    if (!types.Any())
    {
      var name = typeof(T).Name;
      if (element["category"] is string category && !string.IsNullOrWhiteSpace(category))
      {
        name = category;
      }

      appObj.Update(
        status: ApplicationObject.State.Failed,
        logItem: $"Could not find any loaded family to use for category {name}."
      );

      return default;
    }
```